Libraries/LibJS/Runtime/Value.cpp

```diff
@@ -417,6 +419,8 @@ Utf16String Value::to_utf16_string_without_side_effects() const
     case EMPTY_TAG:
         return "<empty>"_utf16;
     default:
+        if (is_cell() && !is_object())
```

Comment:
The `!is_object()` check is unnecessary as we would have taken the `case OBJECT_TAG` branch above.